keystrokes/src/tests/components/Game/Game.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
this doesnt work atm im just leaving it in as reference for later. same with other .test file.